microsoft-edge/webview2/concepts/webview-features-flags.md

```diff
@@ -106,8 +106,11 @@ The following are some of the flags we've seen used.
 | `msAbydosGestureSupport` | Allows users to use gestures (such as the scratchout gesture) to delete text by using a pen.  Valid only if the `msAbydos` flag is enabled. |
 | `msAbydosHandwritingAttr` | Whether the "handwriting-to-text" experience is enabled for input elements at the DOM level.  Valid only if the `msAbydos` flag is enabled. |
 | `msEdgeFluentOverlayScrollbar` | Force-enables Fluent Overlay scrollbars, overriding whatever value `kFluentScrollbar` may hold at the moment. |
+| `msEdgeDevToolsWdpRemoteDebugging` | Enables remote debugging. |
 | `msEnhancedTextContrast` | Improves text contrast enhancement and gamma correction to match the quality and clarity of other native Windows applications.  When this flag is used, font rendering respects user ClearType Tuner settings when applying text contrast enhancement and gamma correction. |
 | `msEnhancedTrackingPreventionEnabled` | Enables native privacy protection features, such as blocking cookies and web requests that reside in domains that are known to be tracking domains. |
+| `msFloatyMode` | Use this to disable Floaty feature, as WebView doesn't support to browser retention experiments. |
+| `msFloatyShouldHonorIndiaHoldout` | Use this to disable Floaty India holdout feature that enables if the user is part of control of the India holdout group, as WebView doesn't support browser retention experiments. |
```

Comment:
"the India holdout group" - is this flag Microsoft internal-only?  @monica-ch , cc @captainbrosset